docs/N+1.md

```diff
@@ -175,23 +175,27 @@ If you run the query, at first you will observe a lot of duplicate requests are
 
 ![Duplicate Upstream Calls](../static/images/docs/n+1-duplicate.png)
 
-This happens because of the 100 posts, a lot them are authored by the same user and by default Tailcall will make a request for every user when requested. You can fix this by setting [dedupe](/docs/directives.md#dedupe) to `true` in [server](/docs/directives.md#server-directive).
+This happens because of the 100 posts, a lot them are authored by the same user and by default Tailcall will make a request for every user when requested.
+You can fix this by setting [dedupe](/docs/directives.md#dedupe) to `true` in [`@http`](/docs/directives.md#http-directive), [`@grpc`](/docs/directives.md#grpc-directive), [`@graphQL`](/docs/directives.md#graphql-directive) or [`@call`](/docs/directives.md#call-directive).
```

Comment:
I think we should only talk about how to fix the dedupe in this case which is via http